contract/build.gradle

```diff
@@ -0,0 +1,53 @@
+apply plugin: 'kotlin'
+apply plugin: 'kotlin-jpa'
+apply plugin: 'kotlin-allopen'
+apply from: '../deterministic.gradle'
+apply plugin: 'net.corda.plugins.cordapp'
+apply plugin: 'maven-publish'
+apply plugin: 'idea'
+
+cordapp {
+    info {
+        vendor = 'Cordapp Supplier'
+    }
+}
+
+dependencies {
+    cordaCompile "net.corda:corda-serialization-deterministic:$corda_release_version"
+    cordaCompile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+    cordaCompile "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
+    compileOnly "org.hibernate:hibernate-core:$hibernate_version"
+    testImplementation "org.jetbrains.kotlin:kotlin-test-junit"
+    testImplementation "junit:junit:$junit_version"
+}
+
+allOpen {
```

Comment:
yeah, likewise for this - if this is an _example_ app do we really expect people to copy/paste all this boilerplate into their app? We really need to build a generic Corda Gradle plugin I guess.